HousingRepairsOnlineApi.Tests/Domain/SorConfigurationExtensionsTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Linq;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.Helpers;
+using Xunit;
+
+namespace HousingRepairsOnlineApi.Tests.Domain
+{
+    public class SorConfigurationExtensionsTests
+    {
+        [Theory]
+        [MemberData(nameof(ValidEarlyExitSorConfigurations))]
+        [MemberData(nameof(ValidSorConfigurationsWithEitherSorCodeOrOptions))]
+        public void GivenValidSorConfiguration_WhenValidating_ThenReturnsTrue(SorConfiguration sorConfiguration)
+        {
+            // Act
+            var actual = SorConfigurationExtensions.IsValid(sorConfiguration);
+
+            // Assert
+            Assert.True(actual);
+        }
+
+        public static TheoryData<SorConfiguration> ValidEarlyExitSorConfigurations()
+        {
+            var result = new TheoryData<SorConfiguration>();
+            foreach (var earlyExitValue in EarlyExitValues.All)
+            {
+                result.Add(new SorConfiguration { Display = "display", Value = earlyExitValue });
+            }
+
+            return result;
+        }
+
+        public static TheoryData<SorConfiguration> ValidSorConfigurationsWithEitherSorCodeOrOptions() =>
+            new()
+            {
+                new SorConfiguration { Display = "display", Value = "value", SorCode = "sor code" },
```

Comment:
I'm wondering if we could use a variable for the repeated values here & then use that variable?

Reply:
Extracted to field constants